src/Service.Host/client/src/components/__tests__/parts/PartTemplate.test.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { cleanup, screen } from '@testing-library/react';
+import render from '../../../test-utils';
+import PartTemplate from '../../parts/PartTemplate';
+
+afterEach(cleanup);
+
+const addItem = jest.fn();
+const updateItem = jest.fn();
+const setEditStatus = jest.fn();
+const setSnackbarVisible = jest.fn();
+const searchProductAnalysisCodes = jest.fn();
+const clearProductAnalysisCodesSearch = jest.fn();
+const history = {
+    push: jest.fn()
+};
+
+const item = {
+    partRoot: 'REACT',
+    description: 'A test',
+    hasDataSheet: 'Y',
+    hasNumberSequence: 'Y',
+    nextNumber: '123',
+    allowVariants: 'Y',
+    variants: 'Integration UNIT',
+    accountingCompany: 'LINN',
+    productCode: 'MECHANICAL',
+    stockControlled: 'Y',
+    linnProduced: 'Y',
+    rmfgCode: 'B',
+    bomType: 'A',
+    assemblyTechnology: 'TH',
+    allowPartCreation: 'Y',
+    paretoCode: 'E',
+    links: { href: '', rel: 'self' }
+};
+
+describe('On Create', () => {
+    beforeEach(() => {
+        render(
+            <PartTemplate
+                editStatus="create"
+                searchProductAnalysisCodes={searchProductAnalysisCodes}
+                clearProductAnalysisCodesSearch={clearProductAnalysisCodesSearch}
+                setSnackbarVisible={setSnackbarVisible}
+                history={history}
+                addItem={addItem}
+                updateItem={updateItem}
+                setEditStatus={setEditStatus}
+                privileges={['part.admin']}
+                itemId={1}
+            />
+        );
+    });
+
+    test('page renders all fields without crashing...', () => {
+        expect(screen.queryByText('PartRoot')).toBeNull();
```

Comment:
What are you testing here?  The test name (renders without crashin) doesn't seem to match the test which is testing whether something is null